R/Simulations-class.R

```diff
@@ -248,18 +251,22 @@ Simulations <- function(fit,
 #' @param sigma2w_est (`numeric`)\cr [`DualSimulations`]
 #' @param fit_biomarker (`list`)\cr see [`DualSimulations`]
 #' @param \dots additional parameters from [`Simulations`]
+#' @param stop_report see [`Simulations`]
```

Comment:
```suggestion
```
same logic, because this will be passed via `...` to the parent class constructor function